intelmq/bots/experts/uwhoisd/expert.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+import socket
+
+from intelmq.lib.bot import Bot
+
+
+class UniversalWhoisExpertBot(Bot):
+    """ Universal Whois expert bot get the whois entry related an a domain,
+    hostname, IP address, or ASN from a centralised uWhoisd instance """
+    server: str = 'localhost'
+    port: int = 4243
+
+    def _whois(self, query: str) -> str:
+        print(query)
```

Comment:
Please remove or replace by `self.logger.debug`

Reply:
woops, that was relic of the debug, I will remove it.